transport/src/main/java/io/netty/channel/pool/SimpleChannelPool.java

```diff
@@ -219,7 +238,7 @@ public void run() {
         return promise;
     }
 
-    private void doReleaseChannel(Channel channel, Promise<Void> promise) {
+    private void doReleaseChannel(final Channel channel, final Promise<Void> promise) {
```

Comment:
This is general code style practice that I try to follow in my projects. It helps to prevent possible bugs. If there's no need to change the value of a variable and someone does, the IDE and/or compiler will provide a warning.
It may also be a helping hint to compiler to perform some optimizations for the code.

I don't mind removing it if in netty projects you don't have such rules.